_posts/2020-01-29-Introduction_to_the_IAC_Team.md

```diff
@@ -0,0 +1,121 @@
+Sometimes, as part of the planning documentation we may need to outline a test plan more formally, but this should ultimately be translated in to RSpec unit and acceptance tests.
+This means that on every code change and PR, we can quickly be alerted to breaking changes.
+ 
+### Prioritisation of Work
+Work is prioritised by a number of factors.
+Product Management will have an influence over a lot of the work we undertake and they have many factors to consider.
+One key component that drives work prioritisation is feedback from the IAC team itself.
+We like to be very interactive with the Puppet Community contributors and understand what they require.
+We will often bring feedback from the community to our stand-ups and more global meetings with Engineering and Product Management.
+Through this feedback we're trying to ensure that the user experience is as close to the requirements of the people using and contributing to the product.
+
+One valuable feedback vector is through the weekly [Office Hours](https://puppet.com/community/office-hours/) sessions.
+Once a week, members from the IAC team will down tools and be available to any queries coming in via Slack or the video conferencing session from Puppet Community contributors.
+If you need help with module development or testing, or you want to determine the best way to do something with a module on the Forge, then this is the place to get that help and send that feedback.
+Quite often, if a limitation or issue in a module has been highlighted here, we will bring this up in our next stand-up meeting, raise a JIRA ticket and ensure that it gets prioritisation for resources.
+There's no better feedback than the people using actively using the module ecosystem and we like to make the most of this valuable resource.
+ 
+ 
+### Types of Work
+Our number one remit and focus is to ensure that those who want to contribute to the Forge have as frictionless an experience as possible.
+That spans everything from a module's lifespan, through initial design / prototyping, to version 1.0.0, right up to the point where it's a veteran module with 100,000s of downloads and a wide and active contributor base.
+Our goal is to develop the tools and processes to ensure that is possible.
+One very important aspect of our job is the Content Triage rotation.
+Two members of the IAC team will take it in turns to review community contributors to Puppet Forge modules.
+We're there to ensure that the change meets the quality standard required but more important, we're there to help contributors understand how to do that.
+We're very fortunate to receive lots of high quality contributions or those willing to act upon feedback.
+The goal is to ensure that the quality of the Puppet modules remains high, whilst constantly being enhanced and reactive to the needs of the community.
+ 
+ 
```

Comment:
Nit: Erroneous newline.

Moreover, this section feels like it could be fleshed out a bit to include working on dev tools and in-house module development, I think? Reading it at face value, we talk high level but don't call it out as much as the triage - though maybe that's on purpose! :)